README.md

```diff
@@ -1,5 +1,23 @@
 [comment]: # ( Copyright Contributors to the Open Cluster Management project )
-# Get started 
+# multicluster-controlplane
+
+The `multicluster-controlplane` is a lightweight Open Cluster Manager (OCM) control plane that is easy to install and has a small footprint. It is more efficient, lightweight, and cost-effective, while improving OCM scalability and support for edge scenarios.
+
+Some of the features are:
+
+- Started the OCM hub control plane in standalone mode.
+- Combined the registration and work agent into a single entity.
+- Reduced the footprint of both the control plane and agent.
+- Integrated the policy into the control plane.
```

Comment:
in this repo, we do not have policy integrated currently. it is under-planning.

Reply:
Removed as suggested. Thanks. 
For this PR, I just took the contents from:
```
https://github.com/open-cluster-management-io/ocm/issues/189
https://github.com/open-cluster-management-io/ocm/issues/99
```
and added a platform compatibility bullet.